loops.js

```diff
@@ -17,8 +21,15 @@
 // loop...
 // klaar!
  // ==========================================
+console.log("Opdracht 2:");
 
-
+for (let i = 0; i < 5; i++) {
+    if (i === 4){
+        console.log("Klaar!");
+    } else {
```

Comment:
keurig geschreven. In het vorige bestand (zie: regel 41) gaf ik aan dat er na de "{" een enter moest. Maar hier heeft Novi hetzelfde als jij - geen enter...